3725/2/controllers/webhooks_controller.rb

```diff
@@ -0,0 +1,28 @@
+require 'telegram/bot'
+require 'open-uri'
+require 'active_support/time'
+require_relative '../lib/commands/check'
```

Comment:
Check this https://stackoverflow.com/questions/735073/best-way-to-require-all-files-from-a-directory-in-ruby